README.md

```diff
@@ -30,8 +30,8 @@ The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsona
 ## Demo Version
 
 A demo version is automatically deployed from our repositories:
-* Deployment from the `master` branch - **[open-event-api.herokuapp.com](http://open-event-api.herokuapp.com/)**
-* Deployment from the `development` branch - **[open-event-api-dev.herokuapp.com](http://open-event-api-dev.herokuapp.com/)**
+* Deployment from the `master` branch - **[api.eventyay.com/open-event-api-dev.herokuapp.com](http://api.eventyay.com/open-event-api.herokuapp.com/)**
```

Comment:
`api.eventyay.com/open-event-api-dev.herokuapp.com` is invalid. Two deployments happen
- api.eventyay.com
- open-event-api.herokuapp.com